lib/loader/umd/index.js

```diff
@@ -173,13 +173,17 @@ var loader = (function(exports) {
   
     function __newString(str) {
       if (str == null) return 0;
-      const length = str.length;
+      const size = str.length << 1;
   
-      const ptr = __new(length << 1, STRING_ID);
+      const ptr = __new(size, STRING_ID);
   
-      const U16 = new Uint16Array(memory.buffer);
+      const buf = new Uint8Array(memory.buffer, ptr, size);
```

Comment:
Is it really faster with `ptr` and `size` in constructor? When I last time checked it was less efficient than original approach

Reply:
Also did you check `TextEncoder` with `utf16-le` for converting string to buffer?